websockify/websocketproxy.py

```diff
@@ -585,7 +587,8 @@ def websockify_init():
                                                  legacy=opts.legacy_syslog)
         syslog_handler.setLevel(logging.DEBUG)
         syslog_handler.setFormatter(log_formatter)
-        logger.addHandler(syslog_handler)
+        root = logging.getLogger()
+        root.addHandler(syslog_handler)
```

Comment:
Shouldn't this be part of #519?

Reply:
Good question, actually this change is required for logging to work correctly with TokenRedis (and token_plugins in general).

In token_plugins.py we create a logger with:
```
    logger = logging.getLogger(__name__)
```
This is the standard way of using logging, in which the logger gets the __name__ of the module (in this case token_plugins), so in case someone is interested the logging level for this module can be tuned independently in the general configuration.

In the case of websocketproxy.py the logging instance used for logging is:
```
logging.getLogger(WebSocketProxy.log_prefix))
```
this is fine, and "WebSocketProxy.log_prefix" is used as the name of the logger, instead of using "__name__" to avoid it having the value "__main__".

The problem is that websocketproxy.py is also where the global logging is configured, so in order to enable also the logging of the other modules of the program we have to configure the root logger of the app. 

If we add the handler just to the "WebSocketProxy.log_prefix" logger then the messages from the other loggers in other modules of the app are lost.

This is the reason I had to update in websocketproxy.py all the places where the global logger is configured to use the root logger and not just the "WebSocketProxy.log_prefix" instance.

In PR519 what I try to address is something different, ie. increasing the logging level from INFO to DEBUG so we get all messages generated from the app (actually not many more, but I will address this in the other PR).